webui/src/default/page-settings.tsx

```diff
@@ -86,6 +86,19 @@ export default function createPageSettings(theme: Theme, prefersDarkMode: boolea
             Extensions for VS Code Compatible Editors
         </Typography>;
 
+    const termsStyle = makeStyles({
+        typography: {
+            marginBottom: theme.spacing(2),
+            fontWeight: theme.typography.fontWeightLight,
+            letterSpacing: 4,
+            textAlign: 'center'
+        }
+    });
+    const downloadTerms: React.FunctionComponent = () =>
+        <Typography classes={{ root: termsStyle().typography }}>
+            By clicking download, you accept this website&apos;s Terms of Use (link to Terms of Use)
```

Comment:
This should link to the term of use page:

By clicking download, you accept this website's [Terms of Use](https://open-vsx.org/terms-of-use).

The font used does not match the rest of the page. I think this should look like this:


![Screenshot 2023-01-12 112359](https://user-images.githubusercontent.com/893942/212123587-250971e8-d93e-41ab-b33a-2df73dba25a6.png)

Reply:
@chrisguindon I've incorporated your feedback.